jaeger-spark-dependencies-cassandra/src/test/java/io/jaegertracing/spark/dependencies/cassandra/CassandraDependenciesJobTest.java

```diff
@@ -53,6 +53,7 @@ public static void beforeClass() {
     cassandraPort = cassandra.getMappedPort(9042);
 
     jaegerCassandraSchema = new GenericContainer<>("jaegertracing/jaeger-cassandra-schema:" + jaegerVersion())
+        .withLogConsumer(frame -> System.out.print(frame.getUtf8String()))
```

Comment:
this will be interleaved with the test logs, making them difficult to read. I would either accumulate these logs in a buffer and print it all at once at the end or at least prefix them with a label like `CASSANDRA: `

Reply:
The test waits for `jaeger-cassandra-schema`, so it will not be interleaved much